core/src/main/resources/application-prod.properties

```diff
@@ -44,3 +44,5 @@ greencity.payment.liq-pay-private-key= ${LIQPAY_PRIVATE_KEY}
 
 #GreenCityClient
 greencity.redirect.green-city-client=https://ita-social-projects.github.io/GreenCityClient/#/ubs/confirm/
+
+greencity.authorization.googleApiKey=${GOOGLE_API_KEY}:default-key
```

Comment:
no need to copy greencity.authorization.googleApiKey=${GOOGLE_API_KEY}:default-key to every profile. once you define it in application.properties - it will be fetched in each profile.